javascript/week1/main.js

```diff
@@ -0,0 +1,98 @@
+// Age-ify (A future age calculator)
+let yearOfBirth = 1988; // it can change so "let" variable
+const yearFuture = 2027; // as the future year given so "const" variable
+let age = yearFuture - yearOfBirth;
```

Comment:
You don't actually need to use `let` for `yearOfBirth` or `age` here as they never change in the code. They can both be `const`

Reply:
I thought 'yearOfBirth' can vary for different users, hence used let. and 'age' will change if 'yearOfBirth' changes . 
otherwise you are right, those should be const.